### DISCUSSION TITLE
Can I add a fixed message to each chatgpt generation?

### DISCUSSION THREAD

**Discussion by Franco990** [n3d1117/chatgpt-telegram-bot#498] [Q&A]
Good evening, I would like to know if it was possible to add a fixed text to each chatgpt response generation

For example 

chatgpt gesture generation 

Created by franco99

Thanks in advance for your support!!

**Comment by Franco990** [comment]
Does anyone know how to do?

**Comment by Jipok** [comment]
Only by changing the code. For example add here:
https://github.com/n3d1117/chatgpt-telegram-bot/blob/4162f8e380ebd558df904b48c9f1d18e933fa256/bot/telegram_bot.py#L771
this:
`response += "Created by " + update.effective_user.first_name`
Which will only work if stream is turned off.
